### ISSUE TITLE
In rapporten "selecteer niets/alles" vinkje om gebieden te selecteren

### ISSUE THREAD

**Issue by joostschouppe** [provinciesincijfers/bugsinjive#54]
Graag hebben we in rapporten dezelfde mogelijkheid als dat je hebt bij het samenstellen van een selectie:

![image](https://user-images.githubusercontent.com/10122639/32673222-9a9cfa84-c64e-11e7-9a9d-74152464e6ed.png)

In een rapport als [dit](https://provincies.incijfers.be/?report=testrapport_vvsg) is het bijzonder lastig om tussen alle gemeenten te moeten zoeken naar degene die je hebt aangezet.

Daarnaast zou het ook heel handig zijn als de selectie "overgeërfd" wordt naar lagere gebiedsniveaus. Dus: als ik Provincie Oost-Vlaanderen heb geselecteerd en naar Gemeenteniveau ga, dan worden alle gemeenten van Oost-Vlaanderen aangezet. Dit is nu zo in de gewone interface.

**Comment by EdwardABF** [comment]
Een overzicht van de gekozen gebieden is inderdaad handig. Maar wil je in een rapport ook alle gebieden kunnen selecteren van de hele provincie? De tweede wens kan ik me dus nog niet goed voorstellen. Misschien kunnen we die nog telefonisch bespreken.

**Comment by joostschouppe** [comment]
Nee, op zich wil ik dat doorgaans niet. Maar ik kan mij wel situaties voorstellen waar ik dat wel wil. Bijvoorbeeld een rapport waar je provincies met elkaar kunt vergelijken.
Het zouden knopjes moeten zijn die je dus zelf aan en uit kunt zetten bij de instellingen betreffende je gebiedskeuze

**Comment by EdwardABF** [comment]
Beide verbeteringen staan op de wensenlijst maar zijn nog niet beoordeeld.

Overzicht van gekozen gebieden in rapport (SWI-66918)

Selectie gebied overnemen bij keuze voor lager gebiedsniveau (SWI-66917)

**Comment by EdwardABF** [comment]
Een overzicht van gekozen gebieden in een rapport is beschikbaar vanaf versie 5.3.0.
